src/screens/FeedbackScreen.tsx

```diff
@@ -0,0 +1,217 @@
+            <Text style={styles.modal_text}>Thank you for your feedback</Text>
+            <Pressable
+              style={styles.modal_button}
+              onPress={() => navigation.navigate(screen_names.HOME)}
+            >
+              <Text style={styles.modal_button_text}>OKAY</Text>
+            </Pressable>
+          </View>
+      </Modal>
+            <TouchableOpacity style={styles.button} disabled={text == "" || selection == ""} onPress={() => setModalVisible(!modalVisible)}>
+                <Text style={styles.button_text}>
+                    SUBMIT FEEDBACK
+                </Text>
+            </TouchableOpacity>
+        </View>
+        </ScrollView>
+    )
+};
+
+type EmojiProp = {
+  emojitype  : {emoji : string , text : string},
+  containerstyle : ViewStyle | ViewStyle[]
+}
+
+const Emojis = (props : EmojiProp) => {
+  const text : string = props.emojitype.text
+  const emoji : string = props.emojitype.emoji
+  return (
+    <View style={props.containerstyle}>
+      <Text style={styles.emoji_image}> {emoji} </Text>
+      <Text style={styles.emoji_text}>{text}</Text>
+    </View>
+    )
+}
+
+type FeedbackStyle = {
+  main_container : ViewStyle,
+  feedback_text : TextStyle,
+  response_container : ViewStyle,
+  emoji_container : ViewStyle,
+  clicked_emoji_container : ViewStyle,
+  response_container_text : TextStyle,
+  emojis_response_container : ViewStyle,
+  textinput_text : TextStyle,
+  button : ViewStyle,
+  button_text : TextStyle,
+  modal_view : ViewStyle,
+  modal_button : ViewStyle,
+  modal_button_text : TextStyle,
+  modal_text : TextStyle,
+  modal_image : ImageStyle,
+  emoji_text : TextStyle,
+  emoji_image : TextStyle
+}
+
+const styles = StyleSheet.create<FeedbackStyle>({
+    main_container : {
+        flex : 1 ,
+        alignItems : "center",
+        backgroundColor : colors.DROIDCONKE_WHITE
+    },
+    feedback_text : {
+        textAlign : "center",
+        color : colors.DROIDCONKE_BLUE ,
+        width : "80%",
+        margin : 30,
+        fontSize : 18,
+        fontFamily : fonts.MONTSERRAT_BOLD
+    },
+    response_container : {
+        alignItems : "center",
+        marginBottom : 30,
+        borderColor : "#F5F5F5",
+        borderWidth : 1,
+        borderRadius : 8,
+        width : "90%"
+    },
+    emoji_container : {
+        height : 67,
+        width : 67,
+        backgroundColor : colors.DROIDCONKE_PEARL,
+        borderRadius : 4 ,
+        margin : 30 ,
+        justifyContent : "center",
+        alignContent : "center",
+    },
+    clicked_emoji_container : {
+      opacity : 0.2
+    },
+    response_container_text : {
+        color : colors.DROIDCONKE_BLACK,
+        fontSize : 14 ,
+        fontFamily : fonts.MONTSERRAT_REGULAR,
+        textAlign : "center",
+        marginTop : 17,
+    },
+    emojis_response_container : {
+        flexDirection : "row",
+        justifyContent : "space-around"
+    },
+    textinput_text : {
```

Comment:
Also in the design, the text input has
- border-radius of 7
- color "#C3C3C3"
- font-family of montserrat_light

You can look into that.